### PR TITLE
Feature/upgrade ionic cli

### PR DESCRIPTION
**Pull request by felipecruz91**
Upgrade the Ionic Quickstarter to use latest version (as of today) of ionic cli, which is v6.13.1

Closes #...
Fixes #...

Tasks: 
- [x] Updated documentation in `docs/modules/...` directory
- [ ] Ran tests in `<quickstarter>/testdata` directory

### CODE REVIEW

**Comment by felipecruz91** [comment]
@bljubisic @michaelsauter Would you expect any issue with this upgrade? Is there an *easy* way for me to test it out?

**Comment by michaelsauter** [comment]
@felipecruz91 The best way is to run the quickstarter tests: https://github.com/opendevstack/ods-core/tree/master/tests#running-the-quickstarter-tests. Manual test would work as well ...

**Comment by bljubisic** [comment]
I don't see any problems with this upgrade, eslint is already at the latest version so everything good

File: fe-ionic/Jenkinsfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ node {
 
 library("ods-jenkins-shared-library@${sharedLibraryRef}")
 
-def ionicVersion = "5.4.16"
+def ionicVersion = "6.13.1"
```
Comment (felipecruz91) [inline fe-ionic/Jenkinsfile:17]: @bljubisic I'm not entirely sure whether this version refers to the Ionic [framework](https://ionicframework.com/docs/reference/release-notes) or just the CLI?

Comment (bljubisic) [reply fe-ionic/Jenkinsfile:17]: I would say for Ionic framework, not just CLI

**Review by bljubisic — DISMISSED** [review]

**Comment by metmajer** [comment]
Thanks @felipecruz91 and @bljubisic 👍

**Comment by clemensutschig** [comment]
is this now done? ....

**Review by bljubisic — APPROVED** [review]

**Pull request merged**